build.gradle.kts

```diff
@@ -81,4 +88,20 @@ tasks.jacocoTestCoverageVerification {
             }
         }
     }
-}
\ No newline at end of file
+}
+
+pmd {
+    isConsoleOutput = true
+    toolVersion = "6.32.0"
+    rulesMinimumPriority.set(5)
+    ruleSets.clear()
+    ruleSetFiles(file("$projectDir/config/pmd/pmd.xml"))
+}
+
+tasks.spotbugsMain {
+    reports.create("html") {
+        isEnabled = true
+        setStylesheet("fancy-hist.xsl")
+    }
+}
+
```

Comment:
`mainClassName = "eu.eutampieri.catacombs.game.Catcombs"` generates a deprecated warning, we should consider changing it with whatever is the non deprecated way of doing this.

Reply:
It is but it comes from Danilo Pianini's example.
The warning doesn't stop the checks from succeding I think